automation/terraform/infrastructure/us-central1.tf

```diff
@@ -22,7 +23,7 @@ locals {
           write_relabel_configs = [
             {
               source_labels: ["__name__"]
-              regex: "(container.*|Coda.*)"
+              regex: "(buildkite.*|container.*|Coda.*)"
```

Comment:
Can you add watchdog.* while you're in here / redeploying the exporter anyway?

Reply:
For sure, will do